### ISSUE TITLE
Explanation on path loss calculation

### ISSUE THREAD

**Issue by cl-cv** [osqzss/gps-sdr-sim#406]
Hello,

I am looking for an explanation on how channel path loss is calculated in the following line:

```c
// Path loss
path_loss = 20200000.0 / rho.d;
```

What is the significance of the constant? Can someone point me to a reference for this calculation?

Thanks

**Comment by osqzss** [comment]
It's the GPS constellation orbital altitude.
